wrap/pybind11/CMakeLists.txt

```diff
@@ -100,7 +100,11 @@ if(CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR)
 else()
   set(PYBIND11_MASTER_PROJECT OFF)
   set(pybind11_system SYSTEM)
-  set(_pybind11_findpython_default OFF)
+  if(CMAKE_VERSION VERSION_LESS "3.18")
```

Comment:
We vendor pybind11 but we don't make changes directly on it. Could you explain why this change is required? If absolutely required this should be submitted upstream or as a VCPKG patch, and we add an option in `wrap` to use system or vcpkg-provided pybind11

Reply:
Sounds good, Just remove changes in `wrap` and comment out the Python VCPKG tests,  and add a TODO. I'll find some time to update `wrap` to allow external Pybind11.

Reply:
done.